src/main/java/frc/utils/AllianceUtilities.java

```diff
@@ -0,0 +1,127 @@
+package frc.utils;
+
+import frc.robot.constants.FieldConstants;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.Timer;
+
+public class AllianceUtilities {
+
+    private static DriverStation.Alliance ALLIANCE = DriverStation.Alliance.Blue;
```

Comment:
Make it equal the default alliance from field constants

Reply:
There's already a private default alliance in DriverStationUtils. Should I change that to FeildContstant's one too?

Reply:
Wait why do you make your own isBlue if there's a function for this in driver station utils?